### ISSUE TITLE
Security Execption in Eclipselink 2.7.4-RC2

### ISSUE THREAD

**Issue by fwgreen** [eclipse-ee4j/eclipselink#361]
```
Exception in thread "main" java.lang.SecurityException: class "javax.persistence.Persistence"'s signer information does not match signer information of other classes in the same package
```

**Comment by lukasj** [comment]
see https://bugs.eclipse.org/bugs/show_bug.cgi?id=525457#c5

**Comment by tuxedo0801** [comment]
I have exactly the same issue, but with 2.7.4 instead of 2.7.4-RC2.
I thought this issue has been fixed with 2.7.4? @lukasj

**Comment by MGwiozda** [comment]
This Issue is not fixed in Version 2.7.4 Can you please remove the javax-folder from the jar?

**Comment by zozidalom** [comment]
The issue still exists for me is there any fix?
